recipes/lazyasd/meta.yaml

```diff
@@ -0,0 +1,55 @@
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set version = "0.1.1" %}
+{% set sha256 = "3be134452b5933918dbbf32809999b2f6786fc17dd3a0dcca328afcf4370e292" %}
+
+package:
+  name: lazyasd
+  version: {{ version }}
+
+source:
+  fn: lazyasd-{{ version }}.tar.gz
+  url: https://github.com/xonsh/lazyasd/archive/{{ version }}.tar.gz
+  # md5 is OK but sha256 is desirable.
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [py2k]
+
+requirements:
+  build:
+    - python
+    # When setuptools is available add the `--single-version-externally-managed --record record.txt` above.
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - lazyasd
+  requires:
+    - pytest
+  commands:
+    - py.test
```

Comment:
We might need to switch to `SRC_DIR` first. Though it is a bit of kludgy hack, it does work and doesn't seem to cause any harm here.